plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -109,6 +110,49 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         }
     }
 
+    @Override
+    public void testAddColumn()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testAddColumn);
+    }
+
+    @Override
+    public void testDropColumn()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testDropColumn);
+    }
+
+    @Override
+    public void testAlterTableAddLongColumnName()
+    {
+        skipSchemaChangeOperationLockInterruptedError(super::testAlterTableAddLongColumnName);
+    }
+
+    @Override
+    protected void testAddAndDropColumnName(String columnName, boolean delimited)
+    {
+        skipSchemaChangeOperationLockInterruptedError(() -> super.testAddAndDropColumnName(columnName, delimited));
+    }
+
+    private void skipSchemaChangeOperationLockInterruptedError(Runnable schemaChangeOperation)
+    {
+        try {
+            schemaChangeOperation.run();
+        }
+        catch (Exception e) {
+            RuntimeException trinoException = getTrinoExceptionCause(e);
+            try {
+                verifyConcurrentAddColumnFailurePermissible(trinoException);
```

Comment:
this is called in `testDropColumn` as well

Reply:
Yes, we override the `testDropColumn` test here, any concern?